modules/@apostrophecms/submitted-draft/ui/apos/components/AposSubmittedDraftAdminBarIcon.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <span v-if="count > 0" class="apos-submitted-draft-admin-bar-container">
```

Comment:
The BEM syntax should be `apos-submitted-draft-admin-bar__container`. Though I think you could shorten to `apos-submitted-drafts__container`.